exporter/datadogexporter/translate_traces_test.go

```diff
@@ -103,12 +105,12 @@ func NewResourceSpansData(mockTraceID [16]byte, mockSpanID [8]byte, mockParentSp
 
 	if resourceEnvAndService {
 		resource.Attributes().InitFromMap(map[string]pdata.AttributeValue{
-			conventions.AttributeContainerID: pdata.NewAttributeValueString("3249847017410247"),
-			conventions.AttributeK8sPod:      pdata.NewAttributeValueString("example-pod-name"),
-			"namespace":                      pdata.NewAttributeValueString("kube-system"),
-			"service.name":                   pdata.NewAttributeValueString("test-resource-service-name"),
-			"deployment.environment":         pdata.NewAttributeValueString("test-env"),
-			"service.version":                pdata.NewAttributeValueString("test-version"),
+			conventions.AttributeContainerID:           pdata.NewAttributeValueString("3249847017410247"),
+			conventions.AttributeDeploymentEnvironment: pdata.NewAttributeValueString("test-env"),
+			conventions.AttributeK8sPod:                pdata.NewAttributeValueString("example-pod-name"),
+			"namespace":                                pdata.NewAttributeValueString("kube-system"),
+			"service.name":                             pdata.NewAttributeValueString("test-resource-service-name"),
+			"service.version":                          pdata.NewAttributeValueString("test-version"),
```

Comment:
nit: use `conventions` constants here, it helps to see which ones are OTel specific and which are not